### ISSUE TITLE
binding model to other model causes Error: Generated byte code contains an operand stack underflow.

### ISSUE THREAD

**Issue by carlosrovira** [apache/royale-asjs#322]
This declaration:

<X:SomeComponent model="{model}"/>

`[INFO] --- royale-maven-plugin:0.9.5-SNAPSHOT:compile-app (default-compile-app) @ royalelogin ---
[INFO] Executing MXMLC in tool group Royale with args: [-load-config=/Users/carlosrovira/Dev/Codeoscopic/Source/sgc/royale/royaleapp/royalelogin/../../../webapp/src/main/webapp/compile-app-config.xml, -source-map=true, -js-output=/Users/carlosrovira/Dev/Codeoscopic/Source/sgc/royale/royaleapp/royalelogin/../../../webapp/src/main/webapp/javascript, -compiler.targets=JSRoyale, /Users/carlosrovira/Dev/Codeoscopic/Source/sgc/royale/royaleapp/royalelogin/src/main/royale/App.mxml]
Error: Generated byte code contains an operand stack underflow.

Error: Generated byte code contains an operand stack underflow.

Error: Generated byte code contains an operand stack underflow.

Error: Generated byte code contains an operand stack underflow.

Error: Generated byte code contains an operand stack underflow.

Error: Generated byte code contains an operand stack underflow.

Error: Generated byte code contains an operand stack underflow.

Error: Generated byte code contains an operand stack underflow.

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------`

**Comment by piotrzarzycki21** [comment]
@carlosrovira is it still valid ?

**Comment by carlosrovira** [comment]
Yes, we still has this issue. If I add for example `<j:Button model="{model}"/>`, I get the same `Error: Generated byte code contains an operand stack underflow.`
